styles/projects.css

```diff
@@ -102,7 +102,151 @@
   margin: 0.75rem 0;
 }
 
+/* Project Detail Popup section */
+.detail-backdrop {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  z-index: 22;
+  background: rgb(193 199 208);
+  background:
+    linear-gradient(
+      127deg,
+      rgb(193 199 208 / 92.2%) 0%,
+      rgb(193 199 208 / 90.8%) 35%,
+      rgb(193 199 208 / 91.4%) 100%
+    );
+  width: 100%;
+  height: 100%;
+  padding: 40px 16px;
+  animation: fade-in 0.45s ease-out;
+}
+
+.detail-popup {
+  display: flex;
+  flex-direction: column;
+  gap: 0.75rem;
+  width: 100%;
+  height: 80vh;
+  padding: 1rem;
+  background-color: #fff;
+  border: 0.0625rem solid #dfe1e6;
+  border-radius: 1rem;
+  overflow: scroll;
+  animation: zoom-in 0.45s ease-out;
+}
+
```

Comment:
Great job team 🤝

- [ ] You may try to make your design responsive enough on your mobile popup screen. Your design is not responsive and it makes parts of the information hidden
- Your design
![Mobile popup](https://user-images.githubusercontent.com/63917725/179836477-460dd67e-c88b-4749-9939-87e6fb1b1a36.PNG)

Reply:
@1GORDON  can kindly let me know which screen-size you used for mobile